content/2015.08.notifications-preamble.md

```diff
@@ -0,0 +1,174 @@
+Title: Notifications in Kinto (Preamble)
+Lang: en
+Url: notifications-kinto-preamble
+Slug: notifications-kinto-preamble
+Date: 2015-08-13
+Summary: Notifications are coming to Kinto! Here's the preamble.
+
+At a recent work week in the lovely French town of [Douarnenez](https://www.qwant.com/?q=Douarnenez&t=images) we laid the groundwork for an upcoming feature in [Kinto](https://kinto.readthedocs.org): *Notifications!*
+
+# tl;dr
+
+For those that are in a hurry, here's the gist: We're going to be integrating a "notifications" system into Kinto with an eye towards simplicity, modernity, and flexibility. Our goal is to provide a solid framework that will allow other systems to take advantage of this new feature without having to jump through too many hoops. If that sounds interesting to you - read on.
+
+# What?
+
+Before we go into the gritty details, let's take a moment to come to an understanding about some core terminology, shall we? Notifications are messages which trigger actions, the combination of which is called an event. Actions can in turn generate further notifications, resulting in further events, and so forth until the end of time (or thread).
+
+Messages can come from *somewhere else*, such as an external service, or can be generated internally. Actions can be native functions of Kinto, local executables, features from plugins, and so forth. In general, actions will produce some sort of interesting output. Events can therefore be thought of as a cycle: a message is received, an action takes place, and a message is generated.
+
```

Comment:
I come late in the party, sorry :|

This section could be a lot clearer I believe. Maybe you could split the introduction comment from the definitions.
And maybe put definitions as bullet points ?
* Notifications: ...
* Actions
* Events
* Messages

Otherwise it's bit hard to follow (at least for non native english)